### DISCUSSION TITLE
Please tell me your favorite baseball team.

### DISCUSSION THREAD

**Discussion by jackofayame** [NAIST-SE/AttractingContributors#1092] [2024 Attraction Contributions]
Please tell me your favorite baseball team. Not only Japanese team but also teams from other countries are fine.
https://github.com/jackofayame/AttractingContributors-Baseball

**Comment by KazumaYamasaki** [comment]
Added to Hanshin Tigers to your README.md
I would appreciate it if you could also make a pull request to [my repository](https://github.com/KazumaYamasaki/AttractingContributors-Keyboard) !
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard

**Comment by jackofayame** [comment]
Thank you for your contribution!
